tools/ocean/argo_getdata.py

```diff
@@ -0,0 +1,158 @@
+# author: Marie Jossé
```

Comment:
can you use the argparse library for that script?

Reply:
Yes I think I can just need to check how it works ;)

Reply:
Is it good now ?